.github/workflows/bench-reports.yml

```diff
@@ -0,0 +1,90 @@
+name: CI Build and Test
```

Comment:
This should use a different name to differentiate between this and the normal CI Build and Test.